src/services/AuthService.js

```diff
@@ -0,0 +1,10 @@
+import axios from './AxiosInstance';
+
+const login = async (email, password) => {
+    return await axios.post('/login', { email, password });
```

Comment:
The await keyword is unnecessary here since the function is already returning a promise. You can remove await to improve performance slightly.
Please check other places
```suggestion
    return axios.post('/login', { email, password });
```